tests/func/test_add.py

```diff
@@ -1013,12 +1015,42 @@ def test_add_to_remote(tmp_dir, dvc, local_cloud, local_remote):
     assert local_remote.hash_to_path_info(hash_info.value).read_text() == "foo"
 
 
+def test_add_to_remote_absolute(tmp_dir, dvc, local_remote):
+    from shortuuid import uuid
+
+    temp_name = uuid()
+    temp_file = os.path.join(gettempdir(), temp_name)
+    with open(temp_file, "w") as stream:
+        stream.write("foo")
```

Comment:
Use `tmp_path` fixtures here? 🙂

Reply:
Isn't `tmp_path` gives the same path as the path of the repo? I need an external path outside of a repo

Reply:
Ah, right. There's `make_tmp_dir` fixture, that should do I think.

Reply:
Do you mean `tmp_path_factory.mktemp`?

Reply:
@isidentical, this one:

https://github.com/iterative/dvc/blob/9909ee85a1f61491a959962477208291e38f8f24/tests/dir_helpers.py#L263

This factory will give you a `tmp_dir` (aka `Path`) like fixture.

Reply:
Thanks!